google_compute_engine/test_compat.py

```diff
@@ -37,3 +37,9 @@
   import unittest2 as unittest
 
 builtin = 'builtins' if sys.version_info >= (3,) else '__builtin__'
+
+# Import `reload` to re-import modules for testing compat.
```

Comment:
Please reword this to: `Import the reload module to re-import modules for testing compat.`